app/common/helpers/collections.py

```diff
@@ -57,13 +57,15 @@ def name(self) -> str:
 
     @property
     def status(self) -> str:
+        submitted = MetadataEventKey.COLLECTION_SUBMITTED in [x.event_key for x in self.collection.collection_metadata]
+
         form_statuses = set(
             [
                 self.get_status_for_form(form)
                 for form in chain.from_iterable(section.forms for section in self.schema.sections)
             ]
         )
-        if {CollectionStatusEnum.COMPLETED} == form_statuses:
+        if {CollectionStatusEnum.COMPLETED} == form_statuses and submitted:
```

Comment:
Small edge case technically slipping through with submitted=True and form_statuses somehow not matching? Do we want to error something in that case?